spec/stored_query_schema.yaml

```diff
@@ -0,0 +1,12 @@
+name: stored_query_schema
+type: object
+required: ['query', 'name']
```

Comment:
Let's add "description" (string) in here, and also add `additionalProperties: false`

Reply:
There's an issue using `additionalProperties: false` here because the API injects `ws_ids` into the query data (see https://github.com/kbase/relation_engine/blob/develop/relation_engine_server/api_versions/api_v1.py#L53 onwards). I don't like it, as it's a bit too "magical", plus it breaks the JSON schema validation if you try to enforce the `additionalProperties` rule. Either the `ws_ids` should be declared in the (relevant) schema files or json schema validation should occur before they get injected.

Reply:
ws_ids is injected into the parameters for the query, but this schema is validating the definition file for the query. It seems like you are talking about having `additionalProperties: false` inside the schema for the params for an individual query, but I'm talking about putting it in the schema for the query definition

Reply:
ah sorry, gotcha.